### PR TITLE
Allow use of OSPRay default renderers

### PR DESCRIPTION
**Pull request by favreau**
And introduces a basic fast BasicRenderer instead of the ExtendedOBJRenderer

### CODE REVIEW

File: brayns/common/material/Material.h [hunk 1]
```diff
@@ -66,7 +66,9 @@ class Material
     BRAYNS_API TextureTypes& getTextures() { return _textures; }
     BRAYNS_API void setTexture(const TextureType& type,
                                const std::string& filename);
-
+    BRAYNS_API void lock() { _locked = true; }
+    BRAYNS_API void unlock() { _locked = false; }
+    BRAYNS_API bool locked() { return _locked; }
```
Comment (tribal-tec) [inline brayns/common/material/Material.h]: const function

File: brayns/common/material/Material.h [hunk 2]
```diff
@@ -66,7 +66,9 @@ class Material
     BRAYNS_API TextureTypes& getTextures() { return _textures; }
     BRAYNS_API void setTexture(const TextureType& type,
                                const std::string& filename);
-
+    BRAYNS_API void lock() { _locked = true; }
+    BRAYNS_API void unlock() { _locked = false; }
+    BRAYNS_API bool locked() const { return _locked; }
```
Comment (tribal-tec) [inline brayns/common/material/Material.h]: please put a doxygen here 'cause I already forgot what this was for again

File: plugins/engines/ospray/OSPRayScene.cpp [hunk 3]
```diff
@@ -991,64 +996,60 @@ uint64_t OSPRayScene::_buildMeshOSPGeometry(const size_t materialId)
 
 void OSPRayScene::commitLights()
 {
-    for (auto renderer : _renderers)
+    size_t lightCount = 0;
```
Comment (tribal-tec) [inline plugins/engines/ospray/OSPRayScene.cpp]: this is never incremented??

Comment (favreau) [reply plugins/engines/ospray/OSPRayScene.cpp]: Fixed

File: brayns/common/material/Material.h (hunk 2 above)
Comment (favreau) [reply brayns/common/material/Material.h]: Doc added

**Comment by favreau** [comment]
Retest this please

**Pull request merged**